### PR TITLE
Ds user stories 31 33

### PR DESCRIPTION
**Pull request by danielpselph**
## All Submissions:

* [X] Merge the target branch into the PR branch. Fix any conflicts that might appear.
* [X] Have you checked to ensure that other pull request are approved for merge [Pull Requests](../../../pulls)?
* [ ] What stories does the PR include
31-33
### New Feature Submissions:

* [X] Have you added an explanation of what your changes do?
* [X] Have you lint your code locally prior to submission?

### Testing:

* [X] Have you written new tests for your core changes, as applicable?
* [X] Have you successfully ran tests with your changes locally?
* [X] All new and existing tests passed.

**Test coverage** = 100%
--------------------------------------------------------------------------------
### Change Log
US31-Created order fulfilled status functionality that allows item_orders to be fulfilled by the merchant changing default order status from pending to packaged.
US32-Added all system orders to admin dashboard with links to order user profile, order id and date created. Need continued work on sorting orders by status.
US33-Added ship function to admin dashboard allowing admin to ship package orders changing order status from packaged to shipped, admin is no longer able to cancel the order once shipped.

-Need to add sort by status function to admin dashboard index

### CODE REVIEW

File: app/controllers/merchant/item_orders_controller.rb [hunk 1]
```diff
@@ -0,0 +1,11 @@
+class Merchant::ItemOrdersController < ApplicationController
+    before_action :require_merchant
+
+    def update
+        @item_order = ItemOrder.find(params[:id])
+        @item_order.fulfill_item_order
+        @item_order.order.package if @item_order.order.item_orders.pluck(:status).all?("fulfilled")
```
Comment (alex-latham) [inline app/controllers/merchant/item_orders_controller.rb:7]: I'm still cracking up about `@item_order.order.item_orders` haha

File: app/views/merchant/orders/show.html.erb [hunk 2]
```diff
@@ -0,0 +1,20 @@
+
+<h1 align = "center">Order #<%= @order.id %></h1>
+<h2 align = "center">Status: <%= @order.status %></h2>
+<center>
+  <table>
+    <tr>
+      <th>Item</th>
+      <th>Status</th>
+      <th>Fullfill Item</th>
```
Comment (alex-latham) [inline app/views/merchant/orders/show.html.erb]: Typo - `Fulfill Item`

Comment (danielpselph) [reply app/views/merchant/orders/show.html.erb]: fixed

**Review by tylertomlinson — APPROVED** [review]

**Review by alex-latham — APPROVED** [review]
Looks good

**Pull request merged**